README.md

```diff
@@ -1,2 +1,555 @@
-# cli
-Clean and elegant CLI development kit.
+# {Name}
+
+[![Build](https://github.com/candy-kingdom/cli/workflows/Build/badge.svg)](https://github.com/JoshuaLight/chalice-restful/actions)
```

Comment:
Have you considered other badges, like code coverage, supported Python versions, etcetera? I'm not too familiar with them, so maybe you could propose other options?

Reply:
I think we should add the `pypi` badge later.

Reply:
Of course! I also see that there is a bug in bade URL: it's from my `chalice-restful` package.

Reply:
Fixed one.